### PR TITLE
Add Zoom link generation instructions

### PR DESCRIPTION
**Pull request by mtwo**
Adding instructions for GC members / TC members / people with Zoom permissions about how to generate unique meeting IDs and how to avoid creating conflicting meetings

@trask FYI

### CODE REVIEW

File: docs/how-to-handle-public-calendar.md [hunk 1]
```diff
@@ -63,6 +63,15 @@ A single Zoom account should not be used for back-to-back meetings or for more t
 You can see which Zoom account any potentially conflicting meetings are using in the meeting descriptions.
 (Note: posting the URLs publicly on GitHub leads to Zoom bombing by random bots).
 
+#### Zoom link generation process
+_This is the process that the person responding to Zoom link creation issues will follow. The instructions under this heading are for project admins (Governance and Technical Committee members, some maintainers) who have access to OpenTelemetry's Zoom account credentials._
+
+1. View the OpenTelemetry meeting calendar, and find your desired time slot, along with the meetings that occur immediately before, during, and after it.
+2. See which OpenTelemetry Zoom accounts are being used for the meetings immediately before, during, and after your desired time. The Zoom account name / number is typically listed in the description of each meeting; if it isn't, you can join a meeting (even if it isn't occuring now), click on the green shield icon in the top left, and see the account name / number in the 'host' field.
```
Comment (mtwo) [inline docs/how-to-handle-public-calendar.md:70]:
```suggestion
2. See which OpenTelemetry Zoom accounts are being used for the meetings immediately before, during, and after your desired time. The Zoom account name / number is typically listed in the description of each meeting; if it isn't, you can join a meeting (even if it isn't occurring now), click on the green shield icon in the top left, and see the account name / number in the 'host' field.
```

File: docs/how-to-handle-public-calendar.md [hunk 2]
```diff
@@ -63,6 +63,15 @@ A single Zoom account should not be used for back-to-back meetings or for more t
...
 
+#### Zoom link generation process
+_This is the process that the person responding to Zoom link creation issues will follow. The instructions under this heading are for project admins (Governance and Technical Committee members, some maintainers) who have access to OpenTelemetry's Zoom account credentials._
+
+1. View the OpenTelemetry meeting calendar, and find your desired time slot, along with the meetings that occur immediately before, during, and after it.
+2. See which OpenTelemetry Zoom accounts are being used for the meetings immediately before, during, and after your desired time. The Zoom account name / number is typically listed in the description of each meeting; if it isn't, you can join a meeting (even if it isn't occurring now), click on the green shield icon in the top left, and see the account name / number in the 'host' field.
+3. Choose a Zoom account that isn't already being used for one of the meetings immediately before, during, or after your desired time slot. You may also choose a Zoom account that is being used *exactly*) once immediately before, during, or after (we can run a maximum of two concurrent meetings with each account).
```
Comment (trask) [inline docs/how-to-handle-public-calendar.md:71]:
```suggestion
3. Choose a Zoom account that isn't already being used for one of the meetings immediately before, during, or after your desired time slot. You may also choose a Zoom account that is being used *exactly* once immediately before, during, or after (we can run a maximum of two concurrent meetings with each account).
```

File: docs/how-to-handle-public-calendar.md [hunk 3]
```diff
@@ -63,6 +63,15 @@ A single Zoom account should not be used for back-to-back meetings or for more t
 You can see which Zoom account any potentially conflicting meetings are using in the meeting descriptions.
 (Note: posting the URLs publicly on GitHub leads to Zoom bombing by random bots).
 
+#### Zoom link generation process
+_This is the process that the person responding to Zoom link creation issues will follow. The instructions under this heading are for project admins (Governance and Technical Committee members, some maintainers) who have access to OpenTelemetry's Zoom account credentials._
```
Comment (trask) [inline docs/how-to-handle-public-calendar.md:67]: Only governance committee members currently have access to the Zoom accounts

```suggestion
_This is the process that the person responding to Zoom link creation issues will follow. The instructions under this heading are for project admins (Governance Committee members) who have access to OpenTelemetry's Zoom account credentials._
```

File: docs/how-to-handle-public-calendar.md [hunk 4]
```diff
@@ -63,6 +63,15 @@ A single Zoom account should not be used for back-to-back meetings or for more t
...
+
+1. View the OpenTelemetry meeting calendar, and find your desired time slot, along with the meetings that occur immediately before, during, and after it.
+2. See which OpenTelemetry Zoom accounts are being used for the meetings immediately before, during, and after your desired time. The Zoom account name / number is typically listed in the description of each meeting; if it isn't, you can join a meeting (even if it isn't occurring now), click on the green shield icon in the top left, and see the account name / number in the 'host' field.
+3. Choose a Zoom account that isn't already being used for one of the meetings immediately before, during, or after your desired time slot. You may also choose a Zoom account that is being used *exactly*) once immediately before, during, or after (we can run a maximum of two concurrent meetings with each account).
+4. Log into that Zoom account, click "Schedule a meeting", and create a recurring meeting with the desired meeting name and no fixed schedule.
+5. Copy the newly generated unique Zoom link and paste it into the calendar invite when you create it.
```
Comment (trask) [inline docs/how-to-handle-public-calendar.md:73]: Maybe mention copying it into both the description and the location fields (I've forgotten to update the location field before)

**Review by trask — APPROVED** [review]

File: docs/how-to-handle-public-calendar.md [hunk 5]
```diff
@@ -63,6 +63,15 @@ A single Zoom account should not be used for back-to-back meetings or for more t
...
+
+1. View the OpenTelemetry meeting calendar, and find your desired time slot, along with the meetings that occur immediately before, during, and after it.
+2. See which OpenTelemetry Zoom accounts are being used for the meetings immediately before, during, and after your desired time. The Zoom account name / number is typically listed in the description of each meeting; if it isn't, you can join a meeting (even if it isn't occurring now), click on the green shield icon in the top left, and see the account name / number in the 'host' field.
+3. Choose a Zoom account that isn't already being used for one of the meetings immediately before, during, or after your desired time slot. You may also choose a Zoom account that is being used *exactly* once immediately before, during, or after (we can run a maximum of two concurrent meetings with each account).
+4. Log into that Zoom account, click "Schedule a meeting", and create a recurring meeting with the desired meeting name and no fixed schedule.
+5. Copy the newly generated unique Zoom link and paste it into the calendar invite when you create it.
```
Comment (mtwo) [inline docs/how-to-handle-public-calendar.md:73]:
```suggestion
5. Copy the newly generated unique Zoom link and paste it into the calendar event's description and location.
```

File: docs/how-to-handle-public-calendar.md (hunk 4 above)
Comment (mtwo) [reply docs/how-to-handle-public-calendar.md]: Added

**Review by svrnm — APPROVED** [review]

**Pull request merged**